Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
Image(filename="dissociation_scheme.png")
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mFileNotFoundError[0m                         Traceback (most recent call last)
Cell [0;32mIn[5], line 1[0m
[0;32m----> 1[0m [43mImage[49m[43m([49m[43mfilename[49m[38;5;241;43m=[39;49m[38;5;124;43m"[39;49m[38;5;124;43mdissociation_scheme.png[39;49m[38;5;124;43m"[39;49m[43m)[49m

File [0;32m/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/IPython/core/display.py:970[0m, in [0;36mImage.__init__[0;34m(self, data, url, filename, format, embed, width, height, retina, unconfined, metadata, alt)[0m
[1;32m    968[0m [38;5;28mself[39m[38;5;241m.[39munconfined [38;5;241m=[39m unconfined
[1;32m    969[0m [38;5;28mself[39m[38;5;241m.[39malt [38;5;241m=[39m alt
[0;32m--> 970[0m [38;5;28;43msuper[39;49m[43m([49m[43mImage[49m[43m,[49m[43m [49m[38;5;28;43mself[39;49m[43m)[49m[38;5;241;43m.[39;49m[38;5;21;43m__init__[39;49m[43m([49m[43mdata[49m[38;5;241;43m=[39;49m[43mdata[49m[43m,[49m[43m [49m[43murl[49m[38;5;241;43m=[39;49m[43murl[49m[43m,[49m[43m [49m[43mfilename[49m[38;5;241;43m=[39;49m[43mfilename[49m[43m,[49m
[1;32m    971[0m [43m        [49m[43mmetadata[49m[38;5;241;43m=[39;49m[43mmetadata[49m[43m)[49m
[1;32m    973[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mwidth [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m [38;5;129;01mand[39;00m [38;5;28mself[39m[38;5;241m.[39mmetadata[38;5;241m.[39mget([38;5;124m'[39m[38;5;124mwidth[39m[38;5;124m'[39m, {}):
[1;32m    974[0m     [38;5;28mself[39m[38;5;241m.[39mwidth [38;5;241m=[39m metadata[[38;5;124m'[39m[38;5;124mwidth[39m[38;5;124m'[39m]

File [0;32m/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/IPython/core/display.py:327[0m, in [0;36mDisplayObject.__init__[0;34m(self, data, url, filename, metadata)[0m
[1;32m    324[0m [38;5;28;01melif[39;00m [38;5;28mself[39m[38;5;241m.[39mmetadata [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[1;32m    325[0m     [38;5;28mself[39m[38;5;241m.[39mmetadata [38;5;241m=[39m {}
[0;32m--> 327[0m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mreload[49m[43m([49m[43m)[49m
[1;32m    328[0m [38;5;28mself[39m[38;5;241m.[39m_check_data()

File [0;32m/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/IPython/core/display.py:1005[0m, in [0;36mImage.reload[0;34m(self)[0m
[1;32m   1003[0m [38;5;250m[39m[38;5;124;03m"""Reload the raw data from file or URL."""[39;00m
[1;32m   1004[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39membed:
[0;32m-> 1005[0m     [38;5;28;43msuper[39;49m[43m([49m[43mImage[49m[43m,[49m[38;5;28;43mself[39;49m[43m)[49m[38;5;241;43m.[39;49m[43mreload[49m[43m([49m[43m)[49m
[1;32m   1006[0m     [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mretina:
[1;32m   1007[0m         [38;5;28mself[39m[38;5;241m.[39m_retina_shape()

File [0;32m/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/IPython/core/display.py:353[0m, in [0;36mDisplayObject.reload[0;34m(self)[0m
[1;32m    351[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mfilename [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:
[1;32m    352[0m     encoding [38;5;241m=[39m [38;5;28;01mNone[39;00m [38;5;28;01mif[39;00m [38;5;124m"[39m[38;5;124mb[39m[38;5;124m"[39m [38;5;129;01min[39;00m [38;5;28mself[39m[38;5;241m.[39m_read_flags [38;5;28;01melse[39;00m [38;5;124m"[39m[38;5;124mutf-8[39m[38;5;124m"[39m
[0;32m--> 353[0m     [38;5;28;01mwith[39;00m [38;5;28;43mopen[39;49m[43m([49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mfilename[49m[43m,[49m[43m [49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_read_flags[49m[43m,[49m[43m [49m[43mencoding[49m[38;5;241;43m=[39;49m[43mencoding[49m[43m)[49m [38;5;28;01mas[39;00m f:
[1;32m    354[0m         [38;5;28mself[39m[38;5;241m.[39mdata [38;5;241m=[39m f[38;5;241m.[39mread()
[1;32m    355[0m [38;5;28;01melif[39;00m [38;5;28mself[39m[38;5;241m.[39murl [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:
[1;32m    356[0m     [38;5;66;03m# Deferred import[39;00m

[0;31mFileNotFoundError[0m: [Errno 2] No such file or directory: 'dissociation_scheme.png'

